operator/deploy/operator.yaml

```diff
@@ -0,0 +1,153 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: pmem-csi
+---
```

Comment:
Do we want to use this as default or should "default" be the default, with selecting a namespace done via kustomize?

Reply:
I chose a non-default namespace just to illustrate how we could limit the operator and all driver resources to a specific namespace. Once we add customize support for this we can make this configurable. If you prefer to keep 'default' namespace I would change.

Reply:
It's okay for now, we'll probably remember to update this when adding kustomize support.